main.go

```diff
@@ -185,10 +185,10 @@ func registerWorkloads(port string, log logger.Handler) {
 
 	log.Info("Registering latest service mesh components for version ", version)
 	// Register workloads
-	for _, manifest := range build.CRDnames {
-		log.Info("Registering for ", manifest)
+	for name, url := range build.CRDnamesURL {
+		log.Info("Registering for ", name)
```

Comment:
The approach is almost same, with a very subtle difference. The difference being generating url using a function from name , instead of having a map[name]url beforehand